### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by derpachu** [derpachu/MI-449-SS17-740-css-stealing-styles-xedvDK#1]
@chrisvfritz Can you take a look at this? It's [hosted here](https://derpachu.github.io/MI-449-SS17-740-css-stealing-styles-xedvDK/) and meets the following criteria:

- [x] The page includes the name of your search engine in big letters (you can make it up), with a search `input` below it, then a button below that to run the search (the button doesn't have to do anything for this prototype)
- [x] The font size of the search engine name is at least 70 pixels, so that it's very easy to see
- [x] `span` elements are wrapped around each letter of the name, like this: `<span>G</span><span>o</span><span>o</span><span>g</span><span>l</span><span>e</span>`, so that they can be styled individually with classes
- [x] The on-hover colors from the paragraphs [on this page](http://skillful-clouds.surge.sh/) are used to color individual letters of your name (similar to the letters in Google's logo)
- [x] Inspect the `input` element at the bottom of the [_Prototyping states_ page of the lesson](5) to create an identical-looking search box for your own site (it should even use the same styles on `focus`)
- [x] The styles from the "Create Account" button [on this page](http://plant-servant.surge.sh/) are used for your "Search" button (it should even use the same styles on `hover` and `active`)
- [x] There are no stylelint errors in your CSS (this will also be expected for all future projects)
- [x] All your CSS exists in a `.css` file

**Comment by KatieMFritz** [comment]
Hi Joe, Chris asked me to take some code reviews for him so he could focus on some new feature development. 😁 

### Class names for CSS
You did a great job finding and applying the different colors to the letters on your site name. 💪  One way you can make your code even stronger is to give your CSS classes more descriptive names. 

Right now, you have `color1`, `color2`, etc. - but reading your code on Github, I can't tell what those colors are, or what your intent is. What would you think about changing the names to describe the actual colors, like `green`, `blue`, `yellow`, etc.? 🌈 

* * *
### Button styles
I like how you inverted the colors on your button - it gives it nice contrast on the white background of your page! ✨ 

The `hover` and `active` states on your button look consistent with the example. However, your button has a `focus` state that gives it an extra border or outline, while the example button doesn't change on `focus`. Can you update yours to match? 
* * *
### Break up long lines
This is an optional recommendation. Your `h1` on [line 13 of your HTML](https://github.com/derpachu/MI-449-SS17-740-css-stealing-styles-xedvDK/blob/master/index.html#L13) is pretty long - it's a little hard to parse visually, and you have to scroll horizontally to see all of it. 

If I were you, I'd break it up into multiple lines, like this:
```html
<h1>
  <span class="green">H</span>
  <span class="blue">O</span>
  ...
</h1>
```
See how it's a lot easier to see how many `span`s/letters there are and what the classes are?
* * *
After you’ve made your changes and pushed them to GitHub and your hosted site, give it a once-over to make sure it looks right, then comment back here and I’ll take another look.

Thanks! :rocket:

**Comment by derpachu** [comment]
made the changes, for the focus on the button, I copied it from the website but I just got rid of it all here

**Comment by KatieMFritz** [comment]
The classes, and spacing both look better! 😁  Very nice commit message, too. 

For that `h1` where you broke it into multiple lines, the `</h1>` should be on its own line as well, properly indented. It makes it easy to see the beginning and end of the element.

For the button, I'm still seeing it change when I apply the `focus` state using Chrome Inspect tools. For the example button, nothing changes at all when I apply `focus`, and I don't see any CSS scoped to the `focus` state of that element. Let me know if you're seeing something different. Otherwise, make your button not change from its normal state to its :focus: state (`active` and `hover` are both good - don't change those).

**Comment by derpachu** [comment]
made the changes

**Comment by KatieMFritz** [comment]
I still see some CSS for `button:focus` in your code, but it doesn't seem to do anything on the page, so I'll say :shipit: . Watch out for unneeded code in the future though! It can be confusing and can sometimes cause conflicts. 🙂

**Comment by derpachu** [comment]
just included to ensure nothing would be generated. I'll add a comment just to clarify.
